x-pack/plugin/esql/qa/server/multi-clusters/src/javaRestTest/java/org/elasticsearch/xpack/esql/ccq/MultiClusterSpecIT.java

```diff
@@ -230,20 +229,13 @@ static CsvSpecReader.CsvTestCase convertToRemoteIndices(CsvSpecReader.CsvTestCas
         }
         int offset = testCase.query.length() - query.length();
         if (offset != 0) {
-            final String pattern = "Line (\\d+):(\\d+):";
+            final String pattern = "\\b1:(\\d+)\\b";
```

Comment:
This was originally replacing the "Line 1:X:" text to the new position after the `FROM index` update. I'm modifying it to replace any `1:X`.
The fix currently works, and I can't think of any obvious case where this will fail in the near future, as `X:Y` is a quite specific string